recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: 33ba671fa85accebbe7ab91a2f4eb575b485ae39396e3273dfbeaef35df948dc
 
 build:
-  number: 0
+  number: 1
```

Comment:
did you bump this?

Reply:
the previous PR was create automatically by  'regro-cf-autotick-bot` and the tests passed.
now in this PR I am adding extra dependencies and changing the tests.

it is done for review.